addons/im_livechat/models/mail_channel.py

```diff
@@ -76,3 +76,11 @@ def cron_remove_empty_session(self):
                 < ((now() at time zone 'UTC') - interval %s)""", ("%s hours" % hours,))
         empty_channel_ids = [item['id'] for item in self.env.cr.dictfetchall()]
         self.browse(empty_channel_ids).unlink()
+
+    @api.model
+    def get_empty_list_help(self, help):
+        if help:
+            action_channel = self.env.ref('im_livechat.mail_channel_action', False)
```

Comment:
because i think we need to do this only when we found this action.
and there is reason to override this action.
Actual problem is this : 
https://drive.google.com/file/d/0B-Gp72uY7uAcZzRJb09PLUVnWk0/view?usp=sharing
In mail module, mail.channel is _inherits with mail.alias', it always call "get_empty_list_help" method of mail.thread, 

inside that method,there is condition, if we add help in action without  oe_view_nocontent_create then it will become true. and that override our help message.
add_arrow = not help or help.find("oe_view_nocontent_create") == -1,  and it will add icons,. so we have to improve this condition then not required to override this
method in im_chatter. and i think this is generic problem..for all models which are inherited
from mail.alias and "oe_view_nocontent_create" css not used in help message.